### ISSUE TITLE
How to use extra loss with Keras model?

### ISSUE THREAD

**Issue by justshot** [HiKapok/tf.extra_losses#4]
Hi

I am using Keras api for model training and testing. May I know how to use the loss function in this project together with Keras api?

Thank you!
